deploy/dependencies/admission-webhook/alertmanager-config-validating-webhook.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  labels:
+    app.kubernetes.io/component: admission-webhook
+    app.kubernetes.io/name: prometheus-operator-admission-webhook
```

Comment:
nit: I noticed some resources don't have the label:
```
    app.kubernetes.io/version: 0.60.0-rhobs1
```
not sure if it was intentional, so just mentioned it